content/en/docs/concepts/workloads/_index.md

```diff
@@ -3,5 +3,55 @@ title: "Workloads"
 weight: 50
 description: >
   Understand Pods, the smallest deployable compute object in Kubernetes, and the higher-level abstractions that help you to run them.
+no_list: true
 ---
 
+{{< glossary_definition term_id="workload" length="short" >}}
+Whether your workload is a single component or several that work together, on Kubernetes you run
+it inside a set of [Pods](/docs/concepts/workloads/pods).
+In Kubernetes, a Pod represents a set of running {{< glossary_tooltip text="containers" term_id="container" >}}
+on your cluster.
+
+A Pod has a defined lifecycle. For example, once a Pod is running in your cluster then
```

Comment:
@sftim, to introduce people to the phases of the Pod lifecycle, would it be useful to list a few and then list your example? It's useful, but it seems as though its jumping straight from lifecycle to how Kubernetes recovers in non-normal lifecycle scenarios. Does what I'm saying make sense?

Reply:
It does make sense - maybe I could make _defined lifecycle_ a hyperlink?

Reply:
Good point.
I think this page should provide a brief introduction to "workloads" and the links to the controller pages.
You could remove the last couple of sentences about the Pod lifecycle (readers can go to the lifecycle page) and instead introduce the workload resources with the next paragraph.
Not sure if this makes things better:

Whether your workload is a single component or several that work together, on Kubernetes you run
it inside a set of [Pods](/docs/concepts/workloads/pods).
A Pod represents a set of running {{< glossary_tooltip text="containers" term_id="container" >}}
on your cluster. Your workload is run in/by containers by/in Pods. Many Pods may be created to keep your
workload running, however, you don't need to manage each Pod directly. Instead, ...